### ISSUE TITLE
ARDUINO HOME SECURITY SYSTEM

### ISSUE THREAD

**Issue by rittikadeb** [prathimacode-hub/IoT-Spot#57]
Hi @prathimacode-hub , I am a GSSOC'22 contributor.
I would like to add the project Arduino Home Security System. I will be using Tinkercad for the simulation and various components like keypad for entering the password, IR sensor for movement detection, alarm, LCD etc.

I would be glad if you could kindly assign me this issue.

**Comment by prathimacode-hub** [comment]
Yup, sure. Just add an issue [template](https://github.com/prathimacode-hub/IoT-Spot/blob/main/.github/issue_template/feature_request.md) using edit option along with details mentioned above. It will be all good to get it assigned. @rittikadeb

**Comment by rittikadeb** [comment]
Hey @prathimacode-hub I edited the issue. Would be glad if you would kindly assign it.

**Comment by prathimacode-hub** [comment]
Issue assigned to @rittikadeb. All the best.
